src/application-insights/azext_applicationinsights/commands.py

```diff
@@ -100,3 +111,10 @@ def load_command_table(self, _):
         g.custom_command('link', 'create_component_linked_storage_account')
         g.custom_command('update', 'update_component_linked_storage_account')
         g.custom_command('unlink', 'delete_component_linked_storage_account')
+
+    with self.command_group('monitor app-insights component continues-export', command_type=export_configurations_sdk, custom_command_type=export_configurations_custom_sdk) as g:
+        g.custom_command('list', 'list_export_configurations')
+        g.custom_show_command('show', 'get_export_configuration')
+        g.custom_command('create', 'create_export_configuration')
+        g.custom_command('update', 'update_export_configuration')
+        g.custom_command('delete', 'delete_export_configuration')
```

Comment:
Usually we add confirmation for 'delete' command to avoid deleting accidentally.